assets/js/translator.js

```diff
@@ -868,9 +874,54 @@ function autoSelectDstLang() {
     }
 }
 
+function dictionaryLookup() {
+    if(modeEnabled('lookup')) {
+        if(currentLookupRequest) {
+            currentLookupRequest.abort();
+        }
+
+        $('#dictionaryLookupResult').empty();
+        var wordToLookup = $('#originalText').val().trim()
+            .split(' ');
+        if(wordToLookup.length <= 3 && wordToLookup[0] !== '') {
```

Comment:
Why 3?

Reply:
@sushain97 
I understand that this limits dictionary lookups for sentences with less than or equal to 3 words, but I don't know why we need this. Could I just remove it?

Reply:
@JPJPJPOPOP If you enter a sentence, you obviously want the entire sentence translation, not dictionary translations for each word. Where to draw the line is the question.